<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcQHubMergeDialog</class>
 <widget class="QDialog" name="lcQHubMergeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge model from CADdrive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ServerLabel">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ProductLabel">
       <property name="text">
        <string>Product</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ServerEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ProductEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="CurrentVersionsLabel">
     <property name="text">
      <string>Current base versions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="CurrentVersionsList">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="AdditionalVersionLabel">
     <property name="text">
      <string>Additional base version</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="AdditionalVersionList"/>
     </item>
     <item>
      <widget class="QLabel" name="AdditionalVersionImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ErrorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcQHubMergeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcQHubMergeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
